benchmarks/rayleigh_taylor_instability_free_surface/kaus_rayleigh_taylor_instability.prm

```diff
@@ -0,0 +1,161 @@
+# Input file for the 2D Rayleigh Taylor benchmark described
```

Comment:
Have you checked if the files still run? Please add a small testcase that includes this .prm file and only runs a single time step on a coarse resolution.

Reply:
yep, I made sure they run (and added DG with limiter) and ran one to completion. It matches the Kaus et al. curve well.